src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -1416,6 +1418,61 @@ def test_aks_nodepool_abort(self, resource_group, resource_group_location):
             self.check('powerState.code', 'Running')
         ])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_machine_cmds(self, resource_group, resource_group_location):
+        aks_name = self.create_random_name('cliakstest', 16)
+        self.kwargs.update({
+            'resource_group': resource_group,
+            'name': aks_name,
+            'ssh_key_value': self.generate_ssh_keys()
+        })
+
+        create_cmd = 'aks create --resource-group={resource_group} --name={name} --enable-managed-identity' \
```

Comment:
add trailing space to distinct different options
```suggestion
        create_cmd = 'aks create --resource-group={resource_group} --name={name} --enable-managed-identity ' \
```